### ISSUE TITLE
Wallet wont sync issue 

### ISSUE THREAD

**Issue by Peter-Dor** [ethereum/mist#3344]
Hey guys,

Im sure you're sick of this question but im really struggling with getting the wallet to sync, ive tried everything from deleting the chain-data and starting geth in --fast --cache=2048 mode but it always struggles to download the last 100 blocks. it starts downloading the chain structure at this point and it never stops it goes well into 7 million blocks with no end in sight.  I've even hard reset my computer (backing everything up on a separate hard drive of course)  to see if that would help. cant find a solution that will work on any site. starting to lose the world to live here really want to take part in the inda hash ICO but cant get access to my account. is there any way of getting a hold of my ether without the wallet e.g. a browser version?   

currently in work so couldn't tell you what version im on i know its the latest one anyway.

```
Version: `0.0.0`
OS & Version: windows
Node version: `geth 0.0.0` 
Number of blocks synchronized: 4,657,000
```

**Comment by dimcoderx** [comment]
Same problem,  i already posted here #2749 , it looks like its an epidemic issue

**Comment by mhogstedt** [comment]
Same issue here. Stops short at around 200 blocks. Balance stays at 0. Cannot transfer any ETH. Spent several hours on this issue now. 

Have learnt a lot of geth console commands in the process though.

**Comment by dimcoderx** [comment]
```
Ethereum Wallet Version: 0.9.3
OS & Version: Ubuntu 17.10
Node version: geth 1.7.3-stable-4bb3c89d
```
I solved this by installing `geth` then i run `$ geth --syncmode "fast"` and i left it for 9 hours, i was 200 blocks behind at this point. At first it started downloading the chain structure i think after 18-19 mln of chain structure it started to download the remaining blocks, then i checked with etherscan.io so i had the latest block, started ethereum wallet and it works (though i haven't done transactions to be sure).

Also later i read, that `--syncmode "fast"` is not safe to use as you "open yourself for attacks" - not sure what it means. now i run just `$ geth`
Also my ethereum folder now is 44 GB, not sure if its full blockchain or if its a light version.

**Comment by mhogstedt** [comment]
Thanks for the help. I never solved it. Used www.myetherwallet.com to get my ether out.

**Comment by Peter-Dor** [comment]
Thanks for the advise guys, never got it working in the end had to import my account to metamask to make transactions. Maybe I'll get it working next year haha

**Comment by TwoChain** [comment]
How do i get my ether out from mist with myetherwallet? I don't see any option? Can i put in my UTC file or password somewhere?

**Comment by Peter-Dor** [comment]
unsure how to do it on myetherwallet, if you download thr metamask plugin for chrome and import your account, the account is in the C drive > users > your login account i.e. pc name > app data (make sure hidden files are shown) > roaming> etheruem> key store. in the keystore file will be your account back it up and save a copy to your desktop. using meta mask import that file using the 'JSON FILE' enter the password you setup on mist and its it you can get access to your ether without the mist app. From there you can send it to your preferred wallet e.h myetherwallet.

Hope that helps

**Comment by Peter-Dor** [comment]
theres the link if you need it https://chrome.google.com/webstore/detail/metamask/nkbihfbeogaeaoehlefnkodbefgpgknn?hl=en

**Comment by TwoChain** [comment]
Yes, that worked. Thank you very much. Finally, I am able to access my ether lol.

**Comment by lorezyra** [comment]
Related: https://github.com/ethereum/mist/issues/2466

Please define OS and version.
